python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -657,6 +657,8 @@ def inner(fn: Callable[..., Any]) -> AssetsDefinition:
                         Nothing,
                         is_required=not (can_subset or asset_spec.skippable),
                         description=asset_spec.description,
+                        code_version=asset_spec.code_version,
```

Comment:
maybe there's a more programmatic way to slurp up any args to `Out` that also exist on `AssetSpec`?